integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesDomainEvents.java

```diff
@@ -15,8 +15,6 @@
 import java.util.Properties;
```

Comment:
update the copyright of ItKubernetesDomainEvents.java
// Copyright (c) 2022, Oracle and/or its affiliates.

Reply:
fixed